app/helpers/helpers.rb

```diff
@@ -0,0 +1,11 @@
+class Helpers
+
+  def self.current_user(session)
```

Comment:
In Sinatra, helpers will typically live in the Application Controller. That way, you don't have to pass the session around.